_layouts/docwithnav.html

```diff
@@ -99,15 +99,38 @@
           </div>
           <div id="tigera-navbar" class="navbar-collapse collapse">
             <ul class="nav navbar-nav navbar-right">
+
```

Comment:
nit: I think there's a corresponding line in RELEASING.md that needs to be updated for this change

> Add TWO new <li> entries to the <span class="dropdown"> in _layouts/docwithnav.html file.

TWO probably goes to ONE now?